### DISCUSSION TITLE
Failed url loading on iOS

### DISCUSSION THREAD

**Discussion by dumcheva-yuliya** [KevinnZou/compose-webview-multiplatform#229] [Q&A]
Hello!
Have this while running on iOS:

> [default] Failed to resolve host network app id to config: bundleID: com.apple.WebKit.Networking instance ID: Optional([_EXExtensionInstanceIdentifier: 9D969F99-4229-4316-8A65-60CA0A1FA2FD])
> [XPC] Handle connection with error: Connection interrupted
> [connection] Connection interrupted: will attempt to reconnect

I also get a white screen in my Webview. However it's fully okay, while running on Android.

Do you have any ideas what I can do?

**Comment by danluciano0814** [comment]
same with me its only white screen on ios but working on android

**Reply by danluciano0814** [reply]
Hi @dumcheva-yuliya, mine is working now after adding size.

```
WebView(
    modifier = Modifier.fillMaxSize().background(Colors.AppColors.White),
    state = state
)
```

**Reply by rishabhsingh2002** [reply]
Not working in my case, can you please help ? 

    WebView(
                state = state,
                modifier = Modifier.fillMaxSize())

**Comment by ienground** [comment]
still same here (18.4)

**Comment by ayushi-sk** [comment]
Same here in 2.0.1

**Reply by ienground** [reply]
I fixed (actually not fix, but I found the other way to use).
Are you using 'compose-cupertino' library and CupertinoTopAppBar?

**Reply by ayushi-sk** [reply]
No, I am using Compose Multiplatform

**Reply by ienground** [reply]
nono it's compose multiplatform and what I talked about is library name. It seems you don't use it zz
